builder/common/generated_data.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "github.com/hashicorp/packer/helper/multistep"
+
+type GeneratedData struct {
```

Comment:
I like this new Package common to all builders.

Do you think it would be a good idea to move this to simply `builder/` ? This way we could see this struct as: `builder.GeneratedData`; because I see a lot of 'common' packages. 

I think it would be nice to add up some comments as well to describe what this struct does/holds; and it what cases it is/should be used.

Nit: A doc for the package would be awesome too !

PS: I like Rakyll's idea on pkg naming:  https://rakyll.org/style-packages/#package-naming

Reply:
Totally agree that there's a lot of common packages already. I think it would be better the way you've suggested!